### ISSUE TITLE
RemoteApp moved around?

### ISSUE THREAD

**Issue by mattrunchey** [lepture/authlib#189]
**Describe the bug**

Previous imports are no longer working.

```python
Python 3.6.4 (default, Jan  8 2020, 14:32:46) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from authlib.flask.client import OAuth
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mrunchey/brokerdev/lib/python3.6/site-packages/authlib/flask/client/__init__.py", line 4, in <module>
    from authlib.integrations.flask_client import OAuth, RemoteApp
ImportError: cannot import name 'RemoteApp'
```

**Comment by phy25** [comment]
https://github.com/lepture/authlib/releases/tag/v0.14

**Comment by mattrunchey** [comment]
> https://github.com/lepture/authlib/releases/tag/v0.14

Thanks, I saw the release.  Why still permit the import and then fail another import inside of the module? 

Are there any more detailed release notes or examples showing where everything got moved? The documentation's second line fails - https://docs.authlib.org/en/latest/#overview .

**Comment by lepture** [comment]
https://docs.authlib.org/en/latest/client/flask.html

```python
from authlib.integrations.flask_client import OAuth
```

**Comment by lepture** [comment]
`authlib.flask.client` is deprecated, and it will be removed in v1.0. I've just fixed the legacy imports in v0.14.1.
